webpack.config.js

```diff
@@ -15,4 +16,15 @@ module.exports = {
       },
     ],
   },
-};
\ No newline at end of file
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+      filename: 'index.html',
+    }),
+  ],
+  devServer: {
+    static: path.join(__dirname, 'dist'),
+    compress: true,
+    port: 8080,
```

Comment:
Hi @Gardimy,

I have reviewed your pull request, and I noticed some issues that need to be addressed before merging it. Please make the following changes:

1. **Empty 'src' folder:** Currently, the 'src' folder is empty, but it is being referenced in the `webpack.config.js` file. As a result, the project fails to build, and the following errors occur:
```
Module not found: Error: Can't resolve './src/index.js' in '/home/t/cr/Leaderboard_project'
ERROR
  Error: Child compilation failed:
  Module not found: Error: Can't resolve '/home/t/cr/Leaderboard_project/src/index.html' in '/home/t/cr/Leaderboard_project'
  ModuleNotFoundError: Module not found: Error: Can't resolve '/home/t/cr/Leaderboard_project/src/index.html' in '/home/t/cr/Lea  derboard_project'
```
Please ensure that the 'src' folder contains the necessary files, such as 'index.js' and 'index.html', or update the `webpack.config.js` file to reflect the correct file paths.

Once you have made these changes, please let me know so that I can re-review the code. Feel free to reach out if you have any questions or need further assistance.

Keep up the good work!

Best regards,
Temitope